ui/app/templates/components/generated-item-list.hbs

```diff
@@ -79,20 +79,17 @@
             data-test-popup-menu-trigger
           />
           <dd.Interactive
-            @text="View {{singularize @itemType}}"
```

Comment:
Extra eyes making sure there's no copy pasta here would be helpful! All of the `@text` args should now be in the yielded block for any `Interactive` elements